pkg/security/agent/client_mocks.go

```diff
@@ -0,0 +1,123 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package agent
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/security/proto/api"
+	"math"
+)
+
+// Mocks
+
+type MockRuntimeSecurityClient struct{}
```

Comment:
Is this generated manually ? Why not using the one generated with `inv -e security-agent.gen-mocks` ?

Reply:
The Runtime Security Client doesn't fit the interface defined in api.SecurityModuleClient (on which the mocks generated by `inv -e security-agent.gen-mocks` are based on). Which is why I created another interface `SecurityModuleClientWrapper`. https://github.com/DataDog/datadog-agent/blob/9e867fd144a1582260014bb939f7dcf5ad35639b/pkg/security/agent/client.go#L21

Reply:
ah yeah alright